tests/query/processors/test_project_rate_limiter.py

```diff
@@ -42,7 +51,7 @@
             binary_condition(
                 ConditionFunctions.IN,
                 Column("_snuba_project_id", None, "project_id"),
-                FunctionCall(None, "array", (Literal(None, 4), Literal(None, 5))),
+                FunctionCall(None, "array", (Literal(None, 3), Literal(None, 5))),
```

Comment:
this change makes no difference right?

Reply:
This change is necessary to have the tests pass. If there is a `=` condition and an `IN` condition, the code does an intersect to find the potential IDs. This worked before because the array wasn't being seen, so only the `=` condition was being considered. Since now both are being used, they have to overlap.

Reply:
cool!